_data/jobs.yml

```diff
@@ -1,3 +1,8 @@
+- expires: 2022-01-28
+  location: University of Oregon, Euegene, OR
+  name: Software Engineer
```

Comment:
The title for the job is "Analyst Programmer" - did they request to have it be Software Engineer?

Reply:
The Google Form contained "Software Engineer" although Ian just this morning commented that sometimes the responder mistakenly puts in their own job title.  The submitter is apparently a professor so I do not believe that's happened in this case.
My opinion is that it's probably OK either way.